ACPs/77-reinventing-subnets/README.md

```diff
@@ -213,167 +289,67 @@ An EVM Subnet may choose to implement this step like so:
-
-The P-Chain nodes must refuse to sign any `SubnetValidatorRegistrationMessage` where the `validationID` does not correspond to an active validator and the `expiry` is in the future.
-
-#### SubnetValidatorWeightUpdateMessage
-
-```text
-+--------------+----------+----------+
-|      codecID :   uint16 |  2 bytes |
-+--------------+----------+----------+
-|       typeID :   uint32 |  4 bytes |
-+--------------+----------+----------+
-| validationID : [32]byte | 32 bytes |
-+--------------+----------+----------+
-|        nonce :   uint64 |  8 bytes |
-+--------------+----------+----------+
-|       weight :   uint64 |  8 bytes |
-+--------------+----------+----------+
-                          | 54 bytes |
-                          +----------+
-```
+There is no requirement of any interaction on the Subnet to issue this transaction. For this transaction to be valid, a signature from the `DisableOwner` specified for this validator in its `RegisterSubnetValidatorTx`. Any remaining $AVAX in the Subnet Validator's `Balance` will be issued to the `RemainingBalanceOwner` defined when this validator was added to the validator set.
 
-- `codecID` is the codec version used to serialize the payload and is hardcoded to `0x0000`
-- `typeID` is the payload type identifier and is `0x00000004` for this message
-- `validationID` is the SHA256 of the `Payload` of the `AddressedCall` in the `RegisterSubnetValidatorTx` adding the validator to the Subnet's validator set
-- `nonce` is the latest nonce stored with the `validationID` on the P-Chain
-- `weight` is the current weight of the Subnet Validator with `validationID`
+The expected path for full removal from a Subnet's validator set is via a `SetSubnetValidatorWeightTx` with weight `0` which requires a Warp message produced by the Subnet. However, the ability to reclaim the balance allotted to a validator without authorization from the Subnet is critical for failed Subnets.
 
-### Managing Subnet Validator Balance
+Note that this does not modify a Subnet's total staking weight. This transaction marks the Subnet Validator as inactive, but does not remove it from the Subnet's validator set. Inactive Subnet Validators can re-activate at any time by increasing their balance with an `IncreaseSubnetValidatorWeightTx`.
 
-#### IncreaseBalanceTx
+Subnet creators should be aware that there is no notion of `MinStakeDuration` that is enforced by the P-Chain. It is expected that Subnets who choose to enforce a `MinStakeDuration` will lock the validator's Stake for the Subnet's desired `MinStakeDuration`.
 
-This transaction can be issued by anybody to add additional $AVAX to the `Balance` for `NodeID` validating `Subnet`. If the Subnet Validator corresponding to `ValidationID` is currently inactive (`Balance` was exhausted or `DisableValidatorTx` was issued), this transaction will move them back to the active validator set.
+#### `IncreaseSubnetValidatorWeightTx`
```

Comment:
Why was this transaction renamed from `IncreaseBalanceTx` to `IncreaseSubnetValidatorWeightTx`? Does it increase the weight of the validator now?

Reply:
Yikes, no this was a typo. Thanks for catching it, should be `IncreaseSubnetValidatorBalanceTx`, to be consistent with the transaction naming of the other new transactions for managing subnet validators.